admin/class-config.php

```diff
@@ -172,6 +99,24 @@ public function localize_media_script() {
 	/********************** DEPRECATED METHODS **********************/
 
 	/**
+	 * Exports the current site's WP SEO settings.
+	 *
+	 * @param bool $include_taxonomy Whether to include the taxonomy metadata the plugin creates.
+	 *
+	 * @return bool|string $return False when failed, the URL to the export file when succeeded.
+	 */
+	public function export_settings( $include_taxonomy ) {
+		_deprecated_function( __METHOD__, 'WPSEO 2.0', 'This method is deprecated, please use the <code>WPSEO_Export</code> class.' );
+
+		$export = new WPSEO_Export( $include_taxonomy );
+		if ( $export->success ) {
```

Comment:
Well not entirely, as I changed how it works. This is part of the deprecated method.